test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -36,96 +44,140 @@ using ::testing::DoAll;
 using ::testing::AtLeast;
 using ::testing::Return;
 
-class OrderingServiceTest : public OrderingTest {
+static logger::Logger log_ = logger::testLog("OrderingService");
+
+class MockOrderingServiceTransport : public network::OrderingServiceTransport {
  public:
-  OrderingServiceTest() {
-    fake_gate =
-        static_cast<ordering::MockOrderingGate *>(gate_transport_service.get());
+
+  void publishProposal(model::Proposal &&proposal,
+                       const std::vector<std::string> &peers) override {
+    publishProposal(proposal, peers);
+  };
+
+  void subscribe(std::shared_ptr<network::OrderingServiceNotification> subscriber) override {
+    subscriber_ = subscriber;
   }
 
-  void SetUp() override {}
+  MOCK_METHOD2(publishProposal,
+               void(const model::Proposal &proposal,
+                    const std::vector<std::string> &peers));
+
+  ~MockOrderingServiceTransport() override = default;
+
+  std::weak_ptr<network::OrderingServiceNotification> subscriber_;
+};
+
+class OrderingServiceTest : public ::testing::Test {
+ public:
+  OrderingServiceTest() {
+    peer.address = address;
 
-  void start() override {
-    OrderingTest::start();
-    client = proto::OrderingService::NewStub(
-        grpc::CreateChannel(address, grpc::InsecureChannelCredentials()));
   }
 
-  void send() {
-    grpc::ClientContext context;
-    google::protobuf::Empty reply;
-    client->SendTransaction(&context, iroha::protocol::Transaction(), &reply);
+  void SetUp() override {
+    wsv = std::make_shared<MockPeerQuery>();
+    fake_transport = std::make_shared<MockOrderingServiceTransport>();
+
   }
 
-  void shutdown() override { OrderingTest::shutdown(); }
 
-  ordering::MockOrderingGate *fake_gate;
-  std::unique_ptr<iroha::ordering::proto::OrderingService::Stub> client;
+  void TearDown() override  {
+    if (loop_thread.joinable())
+      loop_thread.join();
+    }
+
+  ~OrderingServiceTest() = default;
+
+  std::thread loop_thread;
+  std::shared_ptr<MockOrderingServiceTransport> fake_transport;
   std::condition_variable cv;
   std::mutex m;
+  std::string address {"0.0.0.0:50051"};
+  model::Peer peer;
+  std::shared_ptr<MockPeerQuery> wsv;
+
 };
 
+
+TEST_F(OrderingServiceTest, ExampleTest) {
```

Comment:
Name of test doesn't reflect purpose. Also, add description at the first line of the test, that provides steps of tests.